backend/src/index.js

```diff
@@ -22,10 +28,11 @@ io.on( 'connection', socket => {
 
 	const timestamp = new Date();
 
-	socket.on( 'setUser', ( message, reply ) => {
+	socket.on( 'setUser', async( message, reply ) => {
 		const issueKey = `${ message.repoName }:${ message.issueId }`;
-		if ( !db.has( issueKey ) ) {
-			db.set( issueKey, new Map() );
+
+		if ( !await getAsync( issueKey ) ) {
+			await setAsync( issueKey, '{}' );
```

Comment:
Isn't needed.

Reply:
It's needed. Otherwise, you'll get ` Cannot read property 'hasOwnProperty' of null`.

Reply:
See https://github.com/ckeditor/hanghub/pull/16/files#r287737449. We can make only one `get` request and one `set` request to the database. There's no need for updating it in the meantime.